packages/react-native-bridge/index.js

```diff
@@ -71,6 +71,10 @@ export function subscribeMediaUpload( callback ) {
 	return gutenbergBridgeEvents.addListener( 'mediaUpload', callback );
 }
 
+export function subscribeMediaSave( callback ) {
+	return gutenbergBridgeEvents.addListener( 'mediaSave', callback );
+}
+
```

Comment:
Also here, could we add a description of how the MediaSave event should be used? (i.e, what parameters does it expect, etc...

Reply:
👍 added in 08e77ba07